composables/useNft.ts

```diff
@@ -118,17 +118,13 @@ async function getProcessMetadata(nft: NFTWithMetadata) {
     image,
     animationUrl,
     type: metadata.type || '',
-    attributes: getAttributes(),
+    attributes: getAttributes(nft, metadata),
   }
 }
 
 export function getNftMetadata(nft: NFTWithMetadata, prefix: string) {
   // if subsquid already give us the metadata, we don't need to fetch it again
-  if (prefix === 'ahk' || prefix === 'ahp') {
-    return getProcessMetadata(nft)
-  }
-
-  if (nft.meta && nft.meta.image) {
+  if (nft.meta?.image) {
```

Comment:
@preschian 


image and media (animation url) are already present on the NFTEntity

Reply:
seems like sometimes I still got `image: null`. for example on this page https://canary.kodadot.xyz/snek/explore/items?listed=false. at the first query at `nftListWithSearch`